src/molecules/pagination/Pagination.js

```diff
@@ -1,14 +1,23 @@
 // @vue/component
 export default {
   props: {
+    /*
+     * Current page
+     */
     page: {
       type: Number,
       required: true
     },
+    /*
+     * Total size
+     */
     totalSize: {
       type: Number,
       required: true
     },
+    /*
+     * Limit on page
+     */
     limit: {
```

Comment:
Why you added this prop. In my opinion, we just need a total number of pages and current page props